app/hotels/src/map/allHotels/MapView.js

```diff
@@ -184,7 +184,7 @@ export class Map extends React.Component<Props, State> {
 
   renderHotelMarker = (hotel: Object, index: number) => {
     const { selectedIndex } = this.props;
-    const price = hotel?.price;
+    const price = hotel?.money;
```

Comment:
It's weird to read `hotel.money`... Could you maybe rename the field in the query instead?

Reply:
Yeah, good idea, I see that this is weird! I will do it on monday

Reply:
@RobinCsl 🚀